stable/memcached/templates/pdb.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "fullname" . }}

```

Comment:
This resource should have the app, chart, release and heritage labels as in the other resources.